requirements.txt

```diff
@@ -0,0 +1,2 @@
+streamlit>=1.4.0
+folium>=0.11
```

Comment:
Do we need a top-level requirements here vs. having them in `examples` and `tests` where they could conceivably be different?

Whatever the answer is, it should match with setup.py